.github/workflows/all_solutions.yml

```diff
@@ -678,3 +679,52 @@ jobs:
           name: deploy-artifacts
           path: ${{ github.workspace }}\build\BuildArtifacts
           if-no-files-found: error
+
+  create-package-rpm:
```

Comment:
since this job is required for the `run-artifactbuilder`, could you relocate this job the be before the `run-artifactbuilder` ?

Reply:
yeah, we could,, but its not 100% required  i didn't want to rock the boat too much.  I can do that though.